keyboards/cannonkeys/link/rules.mk

```diff
@@ -0,0 +1,2 @@
+SERIAL_DRIVER = vendor
+OLED_DRIVER = ssd1306
```

Comment:
This is the default behaviour, so this file can be removed.

Reply:
Also done - thank you!